packages/http/src/models/Transaction.ts

```diff
@@ -131,44 +224,46 @@ export default class Transaction {
         return this.toAddGatewayV1(json)
       case 'rewards_v1':
         return this.toRewardsV1(json)
+      default:
       case 'poc_receipts_v1':
         return this.toPocReceiptsV1(json)
-      default:
-        return prepareTxn(json)
+      // default:
```

Comment:
Why did you comment this one out?

Reply:
I wanted to check with @allenan if it was ok to do this. Having that rogue object in there breaks my whole scheme of having the classes all extend from `DataModel`

Reply:
so this would assume that by default, any non-specified txn is a poc receipts txn? not sure that would work